### ISSUE TITLE
.git shall be excluded by default

### ISSUE THREAD

**Issue by keradus** [PHP-CS-Fixer/trailing-whitespace-checker#4]
```
ker@dus:~/gitwork/xxx$ ./check-trailing-whitespaces 
Trailing whitespaces detected:
Binary file ./.git/objects/0b/0a8f751d4443008d909062099c13a0760d9c9f matches
Binary file ./.git/objects/0d/6c399df730e4bce4a353a78b56a8b674bd4583 matches
Binary file ./.git/objects/0d/d417e259776a8d967dd4f268fe205178ad5216 matches
Binary file ./.git/objects/13/e2d617defa57d487c02d246eceed5ca2cd1e20 matches
Binary file ./.git/objects/3a/4a343072e6d497b59c428abdec0457e887f412 matches
Binary file ./.git/objects/3c/34d5660763ee1204dfb45c1fa35df25d9d50d9 matches
Binary file ./.git/objects/4c/5e5b5b41cb7caa1850226e8c44e66aae644b38 matches
Binary file ./.git/objects/4c/e61e9abb246babad4529e406df718ad1807505 matches
Binary file ./.git/objects/4f/2322a08d1df1a5fbc5051eb880df1a899f4e9f matches
```

**Comment by SpacePossum** [comment]
yes please :)

**Comment by julienfalque** [comment]
Implemented in #7.
